src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -1583,12 +1583,26 @@ public ModelEventCommand(string modelGuid, string eventName, int value = 1)
                 Value = value;
             }
 
+            [JsonConstructor]
+            public ModelEventCommand(string modelGuid, string eventName)
```

Comment:
Aaron and I had the same thought and even read an article documenting this as a compile time error in 2010, but it compiles fine and so does reach - I think it would be safer to do as sharad suggests and remove the default value though if there is different compile or runtime behavior between mono 328 and .45 

> On Jul 21, 2016, at 3:00 PM, gregmarr <notifications@github.com> wrote:
> 
> In src/DynamoCore/Models/RecordableCommands.cs:
> 
> > @@ -1583,12 +1583,26 @@ public ModelEventCommand(string modelGuid, string eventName, int value = 1)
> >                  Value = value;
> >              }
> >  
> > +            [JsonConstructor]
> > +            public ModelEventCommand(string modelGuid, string eventName)
> public ModelEventCommand(Guid modelGuid, string eventName, int value = 1)
> public ModelEventCommand(Guid modelGuid, string eventName)
> @mjkkirschner How is this not ambiguous when attempting to call the second of these constructors?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>